utils/hare-start

```diff
@@ -30,7 +30,37 @@ die() {
     exit 1
 }
 
+usage() {
+    cat <<EOF
+Usage: $PROG [<option>]...
+
+Starts the cluster.
+
+Options:
+  --node       Starts current node only
```

Comment:
Starts hare and motr services on current node only.

Reply:
Done